client/test/components/list_item.spec.js

```diff
@@ -0,0 +1,127 @@
+// Import Vue and the component being tested
+import { expect } from 'chai'
+import Vue from 'vue'
+import listItem from '../../app/components/list_item.vue'
+
+describe('list_item component', () => {
+  let component;
+  let vm;
+
+  // required component props to prevent Vue validation errors
+  listItem.props.title = 'A component title'
+  listItem.props.link = '#'
+
+
+  afterEach(function() {
+    if (vm && vm._isMounted) vm.$destroy()
+  })
+
+
+  describe('data property', () => {
```

Comment:
Here we have data that should work both if we provide a backgroundCSS value and if not, so I thought useful to test it. But maybe it's discussable. What do you think?